automatic/visualstudiocode/Tools/helpers.ps1

```diff
@@ -0,0 +1,44 @@
+function Set-UpdateChannel() {
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$UpdateChannel
+    )
+
+    $codeSettingsPath = $(Join-Path $env:APPDATA "Code") 
+    if (-not (Test-Path $codeSettingsPath)) {
+        Write-Output "Settings path '$codeSettingsPath' does not exist. Creating it now."
+        New-Item -ItemType Directory -Path $codeSettingsPath | Out-Null
+    } else {
+        Write-Output "Settings path '$codeSettingsPath' already exists."
+    }
+
+    $storageFilePath = $(Join-Path $codeSettingsPath "storage.json")
+    if (-not (Test-Path $storageFilePath)) {
+        Write-Output "Settings file '$storageFilePath' does not exist. Creating it now."
+        New-Item -Path $storageFilePath | Out-Null
+
+        $storageFileContent = @"
+{
+  "updateChannel": "$UpdateChannel"
+}
+"@
+
+        $storageFileContent | Set-Content $storageFilePath
+    } else {
+        Write-Output "Settings file '$storageFilePath' already exists."
+
+        $storageFileContent = ConvertFrom-Json "$(Get-Content $storageFilePath -Encoding UTF8)"
+        try
+        {
+            $storageFileContent.updateChannel = "none"
+            Write-Output "Updated 'updateChannel' to '$UpdateChannel'."
+        }
+        catch
+        {
+            Write-Output "Add new 'updateChannel' node with value '$UpdateChannel'."
+            $storageFileContent | Add-Member -Name "updateChannel" -value $UpdateChannel -MemberType NoteProperty
+        }
+
+        $storageFileContent | ConvertTo-Json -Depth 999 -Compress | Set-Content $storageFilePath
```

Comment:
using compress is wrong, I was just looking through the file and it is uncompressed